lib/bus.js

```diff
@@ -683,18 +683,27 @@ module.exports = function bus(conn, opts) {
                             let propAccessMode = Object.keys(propObj)[0]
                             let propType = propObj[propAccessMode]
 
+                            //get user-defined getter/setters
+                            let propGetter = obj[ifaceName].__lookupGetter__(propName)
+                            let propSetter = obj[ifaceName].__lookupSetter__(propName)
+
+                            //use default getter/setter if not user-defined
+                            if (!propGetter) {
```

Comment:
The check `if (varName == null)` (not the `==` and not the `===`) matches for both `null` and `undefined`. Which is why this is a handy check :-)

As for numbers, the check is still relevant, because `isNaN(null)` returns `false`.